Makefile

```diff
@@ -20,6 +20,8 @@ help:
 	@echo "  update_code       to update the git repository"
 	@echo "  update_deps       to update the python and npm dependencies"
 	@echo "  update_db         to run the database migrations"
+	@echo "  initialize_docker to initialize a docker image"
+	@echo "  update_docker     to update a docker image"
```

Comment:
Yeah. I was on the fence about it, since I couldn't think of a good reason anyone would need to invoke it directly.